app/app_mobile_ios.m

```diff
@@ -3,9 +3,45 @@
 // +build ios
 
 #import <UIKit/UIKit.h>
+#import <UserNotifications/UserNotifications.h>
 
 void openURL(char *urlStr) {
     UIApplication *app = [UIApplication sharedApplication];
     NSURL *url = [NSURL URLWithString:[NSString stringWithUTF8String:urlStr]];
     [app openURL:url options:@{} completionHandler:nil];
-}
\ No newline at end of file
+}
+
+static int notifyNum = 0;
+
+void doSendNotification(UNUserNotificationCenter *center, NSString *title, NSString *body) {
+    UNMutableNotificationContent *content = [UNMutableNotificationContent new];
+    content.title = title;
+    content.body = body;
+
+    notifyNum++;
+    NSString *identifier = [NSString stringWithFormat:@"fyne-notify-%d", notifyNum];
+    UNNotificationRequest *request = [UNNotificationRequest requestWithIdentifier:identifier
+        content:content trigger:nil];
+
+    [center addNotificationRequest:request withCompletionHandler:^(NSError * _Nullable error) {
+        if (error != nil) {
+            NSLog(@"Could not send notification: %@", error);
+        }
+    }];
```

Comment:
I think `content`, `identifier` and `request` need to be released here.
I don't know if exceptions might appear. If so, the release should happen in a `@finally` block.

Reply:
Only content is allocated, fixing.

Reply:
So you’re saying that `[NSString stringWithFormat:@"fyne-notify-%d", notifyNum]` doesn’t allocate an `NSString`? Really?

Reply:
Ah okay … autoreleased. But who releases that if there is no pool? Is this the ARC magic?